en/config_mc/pid_tuning_guide_multicopter.md

```diff
@@ -152,18 +152,33 @@ The following parameters can also be adjusted. These determine the maximum rotat
 
 ### Thrust Curve / Throttle PID Attenuation (TPA) {#thrust_curve}
 
-The tuning above optimises performance around the hover throttle. 
+The tuning above optimises performance around the hover throttle.
 But it can be that you start to see oscillations when going towards full throttle.
 
 There are two ways to counteract that:
+
 - Adjust the **thrust curve** with the [THR_MDL_FAC](../advanced_config/parameter_reference.md#THR_MDL_FAC) parameter (preferred method).
-  The thrust to PWM mapping is linear by default — setting `THR_MDL_FAC` to 1 makes it quadratic. 
-  Values in between use a linear interpolation of the two.
-  Typical values are between 0.3 and 0.5.
-  You can start off with 0.3 and then increase it by 0.1 at a time. 
-  If it is too high, you will start to notice oscillations at lower throttle values. 
+  
+  The mapping from motor control signals (e.g. PWM) to expected thrust is linear by default — setting `THR_MDL_FAC` to 1 makes it quadratic.
+  Values in between use a linear interpolation of the two. Typical values are between 0.3 and 0.5.
+
+  If you have the possibility to _measure_ thrust (e.g. with a [thrust stand](https://www.rcbenchmark.com/pages/series-1580-thrust-stand-dynamometer)),
+  you can measure the relation between the PWM control signal and the motor's actual thrust, and fit a function to the data.
+  [This Notebook][THR_MDL_FAC_Calculation] shows how the thrust model factor `THR_MDL_FAC` may be calculated from previously measured thrust data.
+
+  [![Thrust Curve Compensation](../../images/mc_pid_tuning/thrust-curve-compensation.svg)][THR_MDL_FAC_Calculation]
+
+  > **Note** The mapping between PWM and static thrust depends highly on the battery voltage.
+
+  [THR_MDL_FAC_Calculation]: https://gist.github.com/Finwood/19fe4504fab043d35b5f71bc990e5855
```

Comment:
1. This is not appearing in the gitbook generated page. Is it supposed to be part of the note? Generally prefer not to have links to things outside of control of the library - i.e. that you might delete yourself accidentally. Perhaps put the file in the gitbook tree (I can do s a post process if needed). 
1. What I think you've done here is provided an anchor that is linked to from "This Notebook" above - correct? This works in github, but may not work in gitbook. I will test after this is technical reviewed.

Reply:
1. I could render the file as static html and add it to the gitbook tree if that is the preferred method. Where would I place it?
1. Yes, this is a markdown reference-style link. I've linked from both "This Notebook" and the image itself and didn't want to repeat myself, that's why I did not include inline links.

Reply:
@Finwood 
There isn't a "normal" case for Notebooks. Perhaps though it could go in **px4_user_guide/assets/config/mc/ThrustCurve.ipynb** - then we could still link to the file in the same way as we do for the gist but it would be in the tree.

2.The markdown link works fine. Thanks!